Logger/FileLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string fPath;
```

Comment:
Fields should be named with underscore PascalCase. 
```suggestion
        private string _Path;
```